utils/appliance/__init__.py

```diff
@@ -1617,29 +1613,10 @@ def configuration_details(self):
         except KeyError:
             return None
 
-    def server_id(self):
-        try:
-            return self.configuration_details[2]
-        except TypeError:
-            return None
-
-    def server_region(self):
-        try:
-            return self.configuration_details[0]
-        except TypeError:
-            return None
-
-    def server_name(self):
-        try:
-            return self.configuration_details[1]
-        except TypeError:
-            return None
-
-    def server_zone_id(self):
-        try:
-            return self.configuration_details[3]
-        except TypeError:
-            return None
+    server_id = db_queries.config_detail_method(db_queries.ConfigDetailResult.server_id)
```

Comment:
Right, but I'm wondering if we can get rid of that whole module, ```db_queries```, the configuration details _should_ imho, be "gotten" from the configuration collection of object or something like that.